common/alert/alerts-fr.html

```diff
@@ -0,0 +1,154 @@
+---
+{
+	"title": "GCWeb Alert example",
+	"language": "fr",
+	"altLangPage": "alerts-en.html",
+	"breadcrumbs": [
+		{ "title": "GCWeb home", "link": "https://wet-boew.github.io/GCWeb/index-fr.html" }
+	],
+	"dateModified": "2023-02-28"
+}
+---
+
```

Comment:
To ensure we are conforming to "Language in page"
```suggestion
<p>Cette page nécessite une traduction.</p>
<div lang="en">
```